app/views/devise/registrations/new.html.erb

```diff
@@ -1,54 +1,50 @@
-<div class="container">
+<div class='container'>
 	<div >
-		<article class="card-body mx-auto" style="max-width: 400px;">
-			<h4 class="card-title mt-3 text-center">Registration Form</h4>
-			<form>
-				<%=form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
-					<%= render 'devise/shared/error_messages', resource: resource %>
-					<div class="form-group input-group">
-						<%= f.file_field :avatar, class: "form-control" %>
+		<article class='card-body mx-auto' style='max-width: 400px;'>
```

Comment:
kindly fix the indentation in your vs-code settings. indentation should be of 2 spaces not 4 spaces